core/schema.py

```diff
@@ -805,6 +813,53 @@ def resolve_modules_permissions(self, info, **kwargs):
                 ))
         return ModulePermissionsListGQLType(list(config))
 
+    def resolve_custom_filters(self, info, **kwargs):
+        # TODO - add permission
```

Comment:
Should we create special permission for such graphQL?

Reply:
I don't think it's required, if any restrictions were to be added they should be on specific custom filter level not mutation. Just validating if user exists is enough.

Reply:
ok, thanks for clarification